desc/objectives/_coils.py

```diff
@@ -255,10 +255,11 @@ class CoilLength(_CoilObjective):
     target : float, ndarray, optional
         Target value(s) of the objective. Only used if bounds is None.
         Must be broadcastable to Objective.dim_f. If array, it has to
-        be flattened according to the number of inputs.
+        be flattened according to the number of inputs. Defaults to ``target=2*np.pi``.
```

Comment:
Shouldn't this default depend on the diameter of the coil. What if the minor radius of the last surface is 10 ?

Reply:
We could maybe try to make better default values, but it should be in another PR. This one is just about documenting what is currently there.